### DISCUSSION TITLE
TimescaleDB extension: time_bucket_gapfill()-function

### DISCUSSION THREAD

**Discussion by FlorianStorz** [supabase/supabase#21574] [Questions]
Does the supabase TimescaleDB extension offer the time_bucket_gapfill()-function? 

When using time_bucket() everything works fine but when switching to time_bucket_gapfill() I get the error: 

function "time_bucket_gapfill" is not supported under the current "apache" license

Are there plans to include this function to the TimescaleDB extension in the future?

**Comment by GaryAustin1** [comment]
This would all be up to TimescaleDB.
They limit what can be run by a hosting provider.
https://github.com/supabase/supabase/issues/12342
